src/card.css

```
@@ -0,0 +1,86 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+  border: 1px solid var(--bg-color-main);
+}
+
+.card__image {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  height: 134px;
+  margin-bottom: 40px;
+  padding-right: 3px;
+  padding-left: 3px;
```

I'd rather set width explicitly in the case with this image, however
```suggestion
  padding: 0 3px;
```